### DISCUSSION TITLE
如果我创建了多个higress的实例，分不同的ns的话，那么我创建EnvoyFilter是全局的呢还是可以区分开？

### DISCUSSION THREAD

**Discussion by cuisongliu** [alibaba/higress#1412] [Q&A]
如果可以区分不同的实例是根据什么来的？是根据namespace么？

**Comment by CH3CHO** [comment]
EnvoyFilter 有个 `applyTo` 吧？

**Comment by cuisongliu** [comment]
> EnvoyFilter 有个 `applyTo` 吧？

我这里说的是我部署了2套higress，分不同的namespace部署，如果这样的话我EnvoyFilter是根据什么来区分的？

**Comment by johnlanni** [comment]
envoyfilter只会生效当前安装的ns下的
